cmd/legacy_main.go

```diff
@@ -258,8 +258,7 @@ func runCLIApp(c *cli.Context) (err error) {
 		return fmt.Errorf("error resolving flags and configs: %w", err)
 	}
 
-	config.OverrideWithLoggingFlags(mountConfig, flags.LogFormat,
-		flags.DebugFuse, flags.DebugGCS, flags.DebugMutex)
+	cfg.OverrideWithLoggingFlags(newConfig, newConfig.Debug.Fuse, newConfig.Debug.Gcs, newConfig.Debug.LogMutex)
```

Comment:
Seems like intermediate change - how would someone change the format with this change?

Reply:
Discussed offline, `PopulateNewConfigFromLegacyFlagsAndConfig` function gives us new config which already has log-format resolved.